### ISSUE TITLE
media edit interface - show stages that media item is assigned to

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/limelight#246]
the edit page for each media item should show which stages that media item is currently assigned to.
it should be possible to assign or unassign the media item from stages on the media item's edit screen interface.

**Comment by rohrlach** [comment]
I assume there is more to come than just #276

**Comment by helenvarley** [comment]
great, this is now showing up. however there needs to be some interface text here, as tags & stages are listed together without anything to indicate what they are. it's entirely possible that there could be tags and stages of the same name. it's important that we can clearly see what things are.

**Comment by rohrlach** [comment]
Would it be enough to simply have the 'Add' button retain it's label when assignments exist like so:

![screen shot 2017-12-17 at 10 06 09 am](https://user-images.githubusercontent.com/2830313/34080813-f2094310-e311-11e7-9417-275f0185f3ef.png)

**Comment by helenvarley** [comment]
hmm; maybe. but if so then preferably at the beginning of the line rather than the end. imagine if there are 10+ in each & some very similar, i think for our left-to-right trained eyes it would be helpful to signal at the beginning of the line what this is. @vickis any thoughts?

**Comment by rohrlach** [comment]
Perhaps then if we increase the space between the two groups and then put the add button below the group distinct from the existing tabs?

**Comment by helenvarley** [comment]
maybe what would be clearer is simply to have "Tags:" and "Stages:" at the beginning of each line, & then just the green plus button at the end? adding more lines there is going to push everything else further down the screen (which already needs to be improved by making the media item image smaller - no need to have it so big - separate issue i know! ;) )

**Comment by rohrlach** [comment]
Agreed. Okay, o think we're clear on the solution here. I'll assign it to the incoming team when we get them.
